src/main/java/catan/services/impl/PlayServiceImpl.java

```diff
@@ -138,6 +149,17 @@ private void throwDice(GameBean game) {
         game.setDiceThrown(true);
     }
 
+    private void buyCard(GameUserBean gameUser, GameBean game, Map<String, String> returnedParams) throws PlayException, GameException {
```

Comment:
i dont argee that it should be implemented in this story.
1. we doesn't give any resources to player yet, why it should be charged?
2. there are nothing noticed about it in story description.
3. we already have 65th story with the same functionality, why not to implement it there?